### PR TITLE
Update JAAS tutorial

### PR DESCRIPTION
**Pull request by kian99**
Update the JAAS tutorial with improvements based on actually trying to deploy and use the JIMM server.

### CODE REVIEW

File: tutorial/deploy_jimm_microk8s.rst [hunk 1]
```diff
@@ -106,6 +104,48 @@ Now run the following commands to create offers that will be consumed in the nex
...
+a DNS resolver service that can map any IP address to a hostname.  
+
+| This service can map hostnames of the form ``<anything>[.-]<IP Address>.nip.io`` to return simply ``<IP Address>``.
+| E.g. ``magic.127.0.0.1.nip.io`` resolves to ``127.0.0.1``. This service is very useful when working with an IdP locally for testing.
+
+.. note::
+    The same effect can be obtained by editing your ``/etc/hosts`` file but this would require changes on your host system
+    and within various containers. 
+
+To utilise nip.io, get the address of your traefik-public instance and set the ``external_hostname`` config option as below,
```
Comment (babakks) [inline tutorial/deploy_jimm_microk8s.rst:139]: \`\`nip.io\`\` and \`\`traefik-public\`\`.

File: tutorial/deploy_jimm_microk8s.rst [hunk 2]
```diff
@@ -260,6 +309,20 @@ These values are only used internally between JIMM and Juju controllers.
...
+.. code:: bash
+
+    juju run jimm-cert/0 get-ca-certificate --quiet | yq .ca-certificate | sudo tee /usr/local/share/ca-certificates/jimm-test.crt
+    sudo update-ca-certificates
+
+Verify that you can securely connect to JIMM with the following command:
+
+.. code:: bash
+
+    curl https://jimm.test.localhost/debug/info
```
Comment (babakks) [inline tutorial/deploy_jimm_microk8s.rst:324]: Shouldn't this be `test-jimm.localhost` (like the earlier steps)?

**Review by babakks — APPROVED** [review]

**Review by babakks — APPROVED** [review]

**Review by babakks — APPROVED** [review]
By the way, I think we should only update the `custom_wordlist` file for JIMM-specific words.

**Comment by kian99** [comment]
>By the way, I think we should only update the custom_wordlist file for JIMM-specific words.

I was wondering about that, but the spelling errors for `hostname` and `hostnames` went away after I added them to the wordlist so I'm not sure what's the best approach

**Pull request merged**